roles/de-deploy-service/tasks/uncolored.yaml

```diff
@@ -33,3 +33,7 @@
 - include: restart-service.yaml
   vars:
     restart_service_name: "{{service_name}}"
+  environment:
+    DE_TAG: "{{docker.tag}}"
+    DE_ENV: "{{environment_name}}"
+    HOSTNAME: "{{ hostname_i.stdout_lines[0] }}"
```

Comment:
The environment variables for non-zero-downtime deployments are now being set here.